e2e/issues_test.go

```diff
@@ -0,0 +1,81 @@
+// +build e2e
+
+//
+
+package e2e
+
+import (
+	"bytes"
+	"context"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/hashicorp/consul-terraform-sync/config"
+	"github.com/hashicorp/consul-terraform-sync/controller"
+	"github.com/hashicorp/consul-terraform-sync/event"
+	"github.com/hashicorp/consul-terraform-sync/testutils"
+	"github.com/stretchr/testify/require"
+)
+
+// issues GH-236 and GH-239
+func TestServicesRenderRace(t *testing.T) {
+	const NumberOfServices = 100
+
+	srv := testutils.NewTestConsulServer(t,
+		testutils.TestConsulServerConfig{HTTPSRelPath: "../testutils"})
+	defer srv.Stop()
+
+	tempDir := t.Name()
```

Comment:
I like the `t.Name()` a lot more than the current pattern I've been using to name the temporary directory. Will have to steal... :)

Reply:
I stole that from Kim. I like it is as well. :)